src/transform/ThemeTransform.css

```diff
@@ -94,7 +98,7 @@ styles with only the things which need tweaking.
 }
 
 /* caption color */
-/* en > Barack Obama > 793008506 */ 
+/* en > Barack Obama > 793008506 */
```

Comment:
It would be awesome if you could add similar example articles like this for any of the rules you added or modified so it's easy to test where they apply later.